LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015, Cam Findlay
+Copyright (c) 
```

Comment:
I think it's worth putting something that's obviously a placeholder e.g. [YOUR NAME HERE] rather than leaving it empty